### PR TITLE
Adding another recommendation #1327

### PR DESCRIPTION
**Pull request by j0rt3g4**
Adding another recommendation about doing single instance server of SQL instead of multiple on the same server.

I'm on the middle ground of both people
Issue #1327

### CODE REVIEW

**Comment by j0rt3g4** [comment]
Copy review @nenonix @JohanFreelancer9 is required on this one guys. Thank you in advance!

File: SharePoint/SharePointServer/administration/best-practices-for-sql-server-in-a-sharepoint-server-farm.md [hunk 1]
```diff
@@ -44,6 +44,8 @@ To ensure optimal performance for farm operations, we recommend that you install
 ## Configure specific SQL Server settings before you deploy SharePoint Server
 
 To ensure consistent behavior and performance, configure the following options and settings before you deploy SharePoint Server. 
+
+- To mantain simplicity and ease to mantain database, it's recommended to use a single instance over multiple instances on the same server due to performance overheads.
```
Comment (JohanFreelancer9) [inline SharePoint/SharePointServer/administration/best-practices-for-sql-server-in-a-sharepoint-server-farm.md]:
```suggestion
- To maintain simplicity and for ease of database maintenance, we recommend that you use a single instance over multiple instances on the same server due to performance overheads.
```

**Comment by tseward** [comment]
It should read:

"Due to potential performance issues with maintaining multiple SQL instances, we recommend that you use a single instance of SQL Server."

This brings a strong word of caution as to why you would not want to do this, versus simplicity/database maintenance which aren't really a factor.

File: SharePoint/SharePointServer/administration/best-practices-for-sql-server-in-a-sharepoint-server-farm.md (hunk 1 above)
Comment (JohanFreelancer9) [reply SharePoint/SharePointServer/administration/best-practices-for-sql-server-in-a-sharepoint-server-farm.md]: In accordance with tseward's suggestion. Thank you.

**Comment by j0rt3g4** [comment]
@JohanFreelancer9  @tseward Thank you both! it's great! I really like the final result :)

**Pull request merged**